src/main/resources/migration/common/V30__add_staff_code_column_to_staff_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE staff ADD COLUMN staff_code VARCHAR(10) NOT NULL;
```

Comment:
mmh.. guess this will not apply in environments that have data in it

Reply:
We can do either of this:

ALTER TABLE staff ADD COLUMN staff_code VARCHAR(10) NOT NULL DEFAULT 'DEFAULT_CODE';

OR

ALTER TABLE staff ADD COLUMN staff_code VARCHAR(10) NOT NULL;
UPDATE staff SET staff_code = 'DEFAULT_CODE' WHERE staff_code IS NULL;